cmd/minikube/cmd/start.go

```diff
@@ -171,32 +173,43 @@ func init() {
-	startFlagSet.StringSliceVar(&registryMirror, "registry-mirror", nil, "Registry mirrors to pass to the Docker daemon")
-	startFlagSet.String(openshiftVersion, version.GetOpenShiftVersion(), "The OpenShift version that the minishift VM will run (ex: v1.2.3)")
+	//NEEDINFO: "insecure" is a really inappropriate word, not sure what it's supposed to mean but we really cannot use it.
+	startFlagSet.StringSliceVar(&registryMirror, "registry-mirror", nil, "Registry mirrors to pass to the Docker daemon.")
+	//NEEDINFO: Is this URLs of the mirrors? hostnames? also, there's a blank default and that looks weird, either remove it or populate it
+	startFlagSet.String(openshiftVersion, version.GetOpenShiftVersion(), "The OpenShift version to run. Use the format v<n.n.n>")
 }
 
 // initClusterUpFlags creates the CLI flags which needs to be passed on to 'oc cluster up'
 func initClusterUpFlags() {
 	//clusterUpFlagSet.StringVar(&clusterUpConfig.Image, "image", "openshift/origin", "Specify the images to use for OpenShift")
-	clusterUpFlagSet.Bool(skipRegistryCheck, false, "Skip Docker daemon registry check")
-	clusterUpFlagSet.String(publicHostname, "", "Public hostname for OpenShift cluster")
-	clusterUpFlagSet.String(routingSuffix, "", "Default suffix for server routes")
-	clusterUpFlagSet.Bool(useExistingConfig, false, "Use existing configuration if present")
-	clusterUpFlagSet.String(hostConfigDir, dockerhost.DefaultConfigDir, "Directory on Docker host for OpenShift configuration")
-	clusterUpFlagSet.String(hostVolumesDir, dockerhost.DefaultVolumesDir, "Directory on Docker host for OpenShift volumes")
-	clusterUpFlagSet.String(hostDataDir, "", "Directory on Docker host for OpenShift data. If not specified, etcd data will not be persisted on the host.")
-	clusterUpFlagSet.Bool(forwardPorts, false, "Use Docker port-forwarding to communicate with origin container. Requires 'socat' locally.")
-	clusterUpFlagSet.Int(serverLogLevel, 0, "Log level for OpenShift server")
-	clusterUpFlagSet.StringSliceVarP(&openShiftEnv, openshiftEnv, "e", []string{}, "Specify key value pairs of environment variables to set on OpenShift container")
+	clusterUpFlagSet.Bool(skipRegistryCheck, false, "Skip the Docker daemon registry check.")
+	clusterUpFlagSet.String(publicHostname, "", "Public host name of the OpenShift cluster.")
+	clusterUpFlagSet.String(routingSuffix, "", "Default suffix for the server routes.")
+	clusterUpFlagSet.Bool(useExistingConfig, false, "Use existing configuration if available.")
+	clusterUpFlagSet.String(hostConfigDir, dockerhost.DefaultConfigDir, "Location of the OpenShift configuration files on the host.")
+	clusterUpFlagSet.String(hostVolumesDir, dockerhost.DefaultVolumesDir, "Location of the OpenShift volumes on the host.")
+	clusterUpFlagSet.String(hostDataDir, "", "Location of the OpenShift data on the host. If not specified, etcd data will not be persisted on the host.")
+	//NEEDINFO: what does etcd data mean? persistent?
```

Comment:
This is the cluster state information. Everything which tells OpenShift how to operate. OpenShift cluster communication is built upon _etcd_